src/app/lib/analyticsUtils/index.js

```diff
@@ -194,6 +194,14 @@ export const getAtiUrl = (data = []) => {
     wrap ? `${key}=[${value}]` : `${key}=${value}`,
   );
 
+  const atiReferrer = parsedAtiValues.find(value => value.includes('ref'));
```

Comment:
I'm a bit confused as to what's happening here, would you be able to explain? From what I can see:

- In src/app/containers/ATIAnalytics/atiUrl/index.js, we have added a new beacon value for `ref`, of which the value is `getReferrer`.
- This will then be passed in here as the last element of the `data` array.
- If the referrer value is falsy, it will be stripped from the values.
- If there has been a referrer added, it will be parsed.

What's happening in this block of code afterwards? We are finding if one of the parsed values contains the string 'ref'. If it does, we remove it from the parsed values, and then add it in again at the end? Is it not enough that we have passed it in as the final element in the array?

Reply:
Added the block to ensure that if any other object is added to the array of beacons after the ref object, then the the ref object will always be the last one

Reply:
Opted for this approach as opposed to relying on the ref element always being the last one in the beacons array

Reply:
Reverted changes to `getAtiUrl()` in favour of adding test cases to ensure that the ref parameter is always last.